spec/DigitalOceanV2/Api/DropletSpec.php

```diff
@@ -289,18 +348,49 @@ public function it_returns_the_created_droplet_entity_with_ssh_keys($adapter)
             ->shouldReturnAnInstanceOf('DigitalOceanV2\Entity\Droplet');
     }
 
-    public function can_create_multiple_droplets_at_the_same_time($adapter)
+    /**
+     * @param \DigitalOceanV2\Adapter\AdapterInterface $adapter
+     */
+    public function it_can_create_multiple_droplets_at_the_same_time($adapter)
```

Comment:
Not sure if this test was intentionally left in a state where it wouldn't run but I tried to fix it

Reply:
Please try and keep this PR atomic please, so we can merge fixes we need, without coupling everything to this PR. :)

Reply:
Sorry about that, will try to keep PR more atomic from now on